.travis.yml

```diff
@@ -29,6 +29,7 @@ script:
   - export RUST_BACKTRACE=1
   - export CARGO_INCREMENTAL=1
   - export PATH="$PATH":~/.cargo/bin
+  - cargo test --release --all-features
```

Comment:
Optimally, this should be `--all --all-features`. The `--all` does the operation (test in this case) on all crates in a workspace, while without it it does so only on the top-level one. However, here we have only the top-level one, so it would be only future-proofing. Therefore, I'm not insisting on it. Do you want to change it, or do you want me to merge it as it is?

Reply:
Might as well change it, hold on